### PR TITLE
Adding the auth guide

### PR DESCRIPTION
**Pull request by hklee93**
- Removing the colon from the title for the consistency
- Providing a simple auth guide

### CODE REVIEW

**Comment by fvcope** [comment]
👍

**Comment by aszasz1234** [comment]
Is the access token static or there may have some reason to change?

**Comment by Mehdi5151** [comment]
98

**Review by mooha12 — APPROVED** [review]
minepi.com/mooha120

File: SDK_reference.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Client SDK reference:
```
Comment (tuannguyenphucan1991) [inline SDK_reference.md:1]: 0898717131

Comment (tuannguyenphucan1991) [reply SDK_reference.md:1]: Tuantau

**Comment by hklee93** [comment]
This will be updated soon. There's a pending PR about this.

**Pull request closed without merging**